SU2_CFD/src/output_structure.cpp

```diff
@@ -8883,6 +8892,920 @@ void COutput::SetEquivalentArea(CSolver *solver_container, CGeometry *geometry,
   
 }
 
+void COutput::WriteSurface_Analysis(CConfig *config, CGeometry *geometry, CSolver *FlowSolver) {
```

Comment:
I think this is a great new capability to write out info on surfaces... it looks like they have to be defined as MARKERS within our grid format, correct? We've been thinking about reusing the interpolation routines and fast searches for collecting data at arbitrary surfaces in the solution. Maybe we should discuss how to unify these efforts.